nomad/status_endpoint.go

```diff
@@ -78,6 +78,31 @@ func (s *Status) Peers(args *structs.GenericRequest, reply *[]string) error {
 	return nil
 }
 
+// RPCServers is used to get all the RPC server addresses in a region
+func (s *Status) RPCServers(args *structs.GenericRequest, reply *[]string) error {
```

Comment:
Would it make sense to future-proof this a bit by wrapping the `[]string` with a typical RPC response struct? (onto which we could add more fields later). It'd also help not have pointer-to-slice which always feels weird.

Reply:
Yeah, totally. I'd copied it right out of `Status.Peers` but signatures like those are terrible. :grin: Will fix.